[oracle] Prefetch memory variable changed to 32-bit and default raised to 8MB

rldhont opened the pull request:
## Description
Speeds up oracle table access by orders of magnitude

(cherry-pick from commit c3e7632)

Author :     @bjornharrtell 
Date :       Mon Oct 29 21:27:46 2018 +0100

## Checklist

- [x] Commit messages are descriptive and explain the rationale for changes
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTING.md#contributing-to-qgis) before each commit

bjornharrtell committed:
[oracle] Prefetch memory variable changed to 32-bit and default raised to 8MB

Speeds up oracle table access by orders of magnitude

(cherry-pick from commit c3e7632)

Author :     Björn Harrtell <bjorn@wololo.org>
Date :       Mon Oct 29 21:27:46 2018 +0100

bjornharrtell commented:
Not critical, but might want to consider also cherry-picking https://github.com/qgis/QGIS/pull/8390 which sanitizes and removes some warnings introduced by the initial commit.

bjornharrtell committed:
[oracle] Remove meaningless conditional and improve user input handling

(cherry-pick from commit 4ab377b)

Author :     Björn Harrtell <bjorn@wololo.org>
Date :       Wed Oct 31 14:45:37 2018 +0100

jef-n reviewed (approved):
LGTM

rldhont commented:
@bjornharrtell and @jef-n I have cherry-picked the commit 4ab377b
